client/tests/integration/permissions/runtime_validators.rs

```diff
@@ -0,0 +1,52 @@
+#![allow(clippy::restriction)]
+
+use std::fs;
+
+use eyre::{Context as _, Result};
+use iroha_data_model::{
+    permission::{validator, Validator},
+    prelude::*,
+    transaction::WasmSmartContract,
+};
+use test_network::*;
+
+#[test]
+// TODO: Maybe activate with a feature,
+// which will be enabled in the build-script when `nightly` is enabled?
+// Same for `mint_nft_for_every_use_every_1_sec()` test.
+#[ignore = "Only on nightly"]
+fn deny_new_validators() -> Result<()> {
```

Comment:
@appetrosyan , I need your opinion on this

Reply:
Yeah, I didn't know about nightly toolchain by default until yesterday.
So we will keep it ignored until we fully start using nigthly?